Modules/HR/Database/Migrations/2022_07_15_115016_hr_channels_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class HrChannelsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('hr_channels_table', function (Blueprint $table) {
```

Comment:
```suggestion
        Schema::create('hr_channels', function (Blueprint $table) {
```

We don't use table as suffix in table name.